### PR TITLE
Made some minor changes to event Gallery Page

### LINKED ISSUE #13: Event Gallery
### Stage 1
- [x] Hamburger with navbar
- [x] Cards to show previous events
    - [x] Clicking on the card shows details about the event
- [x] Footer

### Stage 2
- [x] Add all events to events list page
- [x] Add content to every event page

### PR DESCRIPTION
**Pull request by suneel87**
Signed-off-by: Suneel Nadipalli <nsuneel89@gmail.com>

Fixes #13 
-Removed the other template events (only 3 have been left in)
-Replaced template images of remaining 3 events with images(compressed) from college
-Removed the "Load More" button at the bottom

### CODE REVIEW

**Comment by suneel87** [comment]
@aravindvnair99 @VinayakTekade I've made the requested changes in the Event Gallery Page

**Comment by Subaharish** [comment]
@suneel87 Pictures are not added in the individual blog pages.

**Comment by suneel87** [comment]
@Subaharish @aravindvnair99 @VinayakTekade I have added the pictures for the individual blog pages

**Review by VinayakTekade — APPROVED** [review]

File: functions/views/eventGallery.ejs [hunk 1]
```diff
@@ -136,7 +136,7 @@
 							</div>
 						</a>
 					</div>
-					<!-- blog-item-4 -->
+					<!--&lt;!&ndash; blog-item-4 &ndash;&gt;
```
Comment (aravindvnair99) [inline functions/views/eventGallery.ejs:139]:
```suggestion
					<!--
```

File: functions/views/eventGallery.ejs [hunk 2]
```diff
@@ -164,7 +164,7 @@
 						</a>
 					</div>
 
-					<!-- blog-item-5 -->
+					&lt;!&ndash; blog-item-5 &ndash;&gt;
```
Comment (aravindvnair99) [inline functions/views/eventGallery.ejs:167]:
```suggestion
```

File: functions/views/eventGallery.ejs [hunk 3]
```diff
@@ -192,7 +192,7 @@
 						</a>
 					</div>
 
-					<!-- blog-item-6 -->
+					&lt;!&ndash; blog-item-6 &ndash;&gt;
```
Comment (aravindvnair99) [inline functions/views/eventGallery.ejs:195]:
```suggestion
```

File: functions/views/eventGallery.ejs [hunk 4]
```diff
@@ -219,7 +219,7 @@
 						</a>
 					</div>
 
-					<!-- blog-item-7  -->
+					&lt;!&ndash; blog-item-7  &ndash;&gt;
```
Comment (aravindvnair99) [inline functions/views/eventGallery.ejs:222]:
```suggestion
```

File: functions/views/eventGallery.ejs [hunk 5]
```diff
@@ -245,7 +245,7 @@
 							</div>
 						</a>
 					</div>
-					<!-- blog-item-8 -->
+					&lt;!&ndash; blog-item-8 &ndash;&gt;
```
Comment (aravindvnair99) [inline functions/views/eventGallery.ejs:248]:
```suggestion
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@suneel87 Don't replace symbols with HTML entities for comments.

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**